<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormChat</class>
 <widget class="QWidget" name="FormChat">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{	
	padding: 0px 8px;
	color: rgb(255, 255, 255);
	selection-background-color: rgba(180, 190, 140, 40);
}

QTextEdit{	
	padding: 4px 2px 0px 8px;
	color: rgb(255, 255, 255);
	selection-background-color: rgba(180, 190, 140, 40);
}

QListView{	
	padding: 8px 8px;
	color: rgb(255, 255, 255);
}
QWidget:disabled{
	color: rgb(120, 120,120);
}
QPushButton:disabled{
	border-image: url(:/stylesheet/button_disabled.png);
}
/*PushButton*/
QPushButton{
	color: rgb(180, 188, 142);
	border-image: url(:/stylesheet/button.png);
}
QPushButton:hover, QPushButton:focus{	
	color: rgb(255, 255, 255);	
	border-image: url(:/stylesheet/button_hover.png);
}
QPushButton:pressed{
	padding-top: 2px;
	border-image: url(:/stylesheet/button_pressed.png);
}
QLabel{
	color: rgb(255, 255, 255);
}

#_te_chat, #_lv_users, #_l_avatar{
	border: 1px solid rgb(50, 50, 50);
	border-radius: 2px;
	background-color: rgba(255, 255, 255, 5);
}


#_le_message{
	border: 1px solid white;
	border-radius: 2px;
	background-color: rgba(255, 255, 255, 5);
}

QScrollBar:vertical {
    border: 1px solid grey;
	background-color: rgba(0, 0, 0, 100);
    width: 20px;
    margin: 22px 0 22px 0;
  }
QScrollBar::handle:vertical {
	border: 1px solid grey;
    min-height: 20px;
	background-color:  rgba(0, 0, 0, 200);
  }

QScrollBar::add-line:vertical {
	border: 1px solid grey;
	background-color: transparent;
	height: 20px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
	bottom:2px;
  }

QScrollBar::sub-line:vertical {
	border: 1px solid grey;
	background-color: rgba(0, 0, 0, 100);
	height: 20px;
	subcontrol-position: top;
	subcontrol-origin: margin;
  }

QScrollBar::down-arrow:vertical {
	width: 15px;
	height: 15px;
	image: url(:/stylesheet/down_arrow.png);	
  }

QScrollBar::up-arrow:vertical{
	width: 15px;
	height: 15px;
	image: url(:/stylesheet/up_arrow.png);	
  }

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/stylesheet/frame_chat.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="_w_widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>754</x>
     <y>0</y>
     <width>260</width>
     <height>748</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>748</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>260</width>
     <height>748</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>39</number>
    </property>
    <property name="rightMargin">
     <number>22</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="_l_avatar">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Constantia</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="_l_rank">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/ranks/rank_1.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>4</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="_l_nick">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Revue BT</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Nick</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/resources.qrc">:/menu/menu_middle_2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="_l_game_wins">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Revue BT</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(207, 207, 207);</string>
        </property>
        <property name="text">
         <string>Juegos ganados</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_l_wins">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Revue BT</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(97, 255, 97);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="_l_game_losses">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>15</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Revue BT</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(207, 207, 207);</string>
        </property>
        <property name="text">
         <string>Juegos perdidos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="_l_losses">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Revue BT</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 65, 68);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/resources.qrc">:/menu/menu_middle.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="_l_users_connected">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Revue BT</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Usuarios conectados</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="_lv_users">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>384</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/resources.qrc">:/menu/menu_middle.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="_pb_exit">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Revue BT</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
        <property name="shortcut">
         <string>Esc</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="_w_widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>726</width>
     <height>748</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>726</width>
     <height>748</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>726</width>
     <height>748</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>18</number>
    </property>
    <property name="topMargin">
     <number>38</number>
    </property>
    <property name="rightMargin">
     <number>24</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="_pb_create_game">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Revue BT</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Crear Partida</string>
        </property>
        <property name="shortcut">
         <string>Alt+C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_pb_join_game">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Revue BT</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Unirse a Partida</string>
        </property>
        <property name="shortcut">
         <string>Alt+U</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/resources.qrc">:/menu/menu_middle.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="_te_chat">
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Constantia'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="_le_message">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>38</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
